### PR TITLE
docs: review documentation

### PR DESCRIPTION
**Pull request by LayZeeDK**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Docs are a bit lacking and inconsistent.

Issue Number: N/A

## What is the new behavior?
Consistent documentation.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information
- Add a packages table for the root readme.
- Remove the duplicate contributors table.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,14 @@
 
 Monorepo for the `ngworker` NPM organization. Packages for Angular applications.
 
+## Packages
+
+Packages in this monorepo.
+
+| Package                                                     | Description                                                                         |
+| ----------------------------------------------------------- | ----------------------------------------------------------------------------------- |
+| [Spectacular (@ngworker/spectacular)](packages/spectacular) | Spectacular Angular testing. Test harnesses for Angular applications and libraries. |
```
Comment (NachoVazquez) [inline README.md:17]: Maybe a link to the Spectacular README?

File: packages/spectacular/README.md [hunk 2]
```diff
@@ -4,43 +4,65 @@
...
 > Spectacular Angular testing.
 
+Spectacular offers test harnesses for Angular applications and libraries.
+
 ## Application testing
 
+Spectacular's application testing API configures the Angular testing module and
+bootstraps a component while ensuring that all application-level hooks are run.
+
+The application test harness is used to test configuration Angular modules,
```
Comment (NachoVazquez) [inline packages/spectacular/README.md:14]:
```suggestion
The application test harness is used to test Angular modules configuration,
```

File: packages/spectacular/README.md [hunk 3]
```diff
@@ -4,43 +4,65 @@
...
-The feature testing API helps configure the Angular testing module and sets up a
-test harness for an Angular feature module. It contains a few companion services
-that wrap Angular's built-in navigation services, but adjust to the Angular
-feature module under test.
+Spectacular's feature testing API configures the Angular testing module and sets
+up a test harness for an Angular feature module. It contains a few companion
+services that wrap Angular's built-in navigation services, but adjusted to the
+Angular feature module under test.
+
+The feature test harness is used to test feature modules and shell modules.
```
Comment (NachoVazquez) [inline packages/spectacular/README.md:32]: This excerpt doesn't mention anything about testing the feature module routing. I think that's an important part of the feature testing if not the most important.

We should emphasize in that.

File: packages/spectacular/README.md [hunk 4]
```diff
@@ -4,43 +4,65 @@
...
+| Export name                              | Kind                                          | Description                                                                                                                 |
+| ---------------------------------------- | --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `createFeatureHarness`                   | Factory for `SpectacularFeatureHarness`       | Configure `SpectacularFeatureTestingModule`, bootstrap `SpectacularAppComponent` and navigate to the default feature route. |
+| `CreateFeatureHarnessOptions`            | Options for `createFeatureHarness`            | Feature harness options.                                                                                                    |
+| `SpectacularFeatureHarness`              | Interface                                     | A harness for testing an Angular feature module.                                                                            |
```
Comment (NachoVazquez) [inline packages/spectacular/README.md:40]: Maybe something like this?
```suggestion
| `SpectacularFeatureHarness`              | Interface                                     | A harness for testing an Angular feature module routing.                                                                            |
```

**Review by NachoVazquez — CHANGES_REQUESTED** [review]

File: packages/spectacular/README.md (hunk 3 above)
Comment (LayZeeDK) [reply packages/spectacular/README.md:32]: That depends on the structure of the routing module. Does it import all its routed component dependencies?

A feature module on the other hand either has routing itself or imports a routing module. It's the entry point from a higher-level route.

File: packages/spectacular/README.md (hunk 2 above)
Comment (LayZeeDK) [reply packages/spectacular/README.md:14]: Either *configuration Angular modules* or *Angular configuration modules*.

File: README.md (hunk 1 above)
Comment (LayZeeDK) [reply README.md:17]: There's already a link to the Spectacular readme:
```
(packages/spectacular)
```

File: packages/spectacular/README.md (hunk 4 above)
Comment (LayZeeDK) [reply packages/spectacular/README.md:40]: There's routing modules and feature modules. Not feature routing modules.

File: packages/spectacular/README.md (hunk 3 above)
Comment (NachoVazquez) [reply packages/spectacular/README.md:32]: I see your point, nonetheless, I think we should make more clear what problem are we solving with these tools or what do we think these are more useful.

Maybe this section is not the place though.

Comment (LayZeeDK) [reply packages/spectacular/README.md:32]: Let's change it to a *routed feature module*.

Comment (NachoVazquez) [reply packages/spectacular/README.md:32]: That sounds better, after all the API expects a `feature-path`.

Comment (LayZeeDK) [reply packages/spectacular/README.md]: Have another look 🙂

**Review by NachoVazquez — APPROVED** [review]

**Pull request merged**